lib/Report/Model/BarChartDataSet.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace PhpBench\Report\Model;
+
+class BarChartDataSet
+{
+    /**
+     * @var string
+     */
+    public $name;
+    /**
+     * @var mixed[]
+     */
+    public $xSeries;
+    /**
+     * @var mixed[]
+     */
+    public $ySeries;
+    /**
+     * @var mixed[]
+     */
+    public $errorMargins;
```

Comment:
If going the pulic property way then change the Table components to do this too (although possibly there is a use case for having richer models when rendering on the console f.e.)